pull/github_membership.go

```diff
@@ -121,3 +121,62 @@ func (mc *GitHubMembershipContext) IsCollaborator(org, repo, user, desiredPerm s
 
 	return perm.GetPermission() == desiredPerm, nil
 }
+
+func (mc *GitHubMembershipContext) ListOrganizationMembers(org string) ([]string, error) {
+	opt := &github.ListMembersOptions{
+		ListOptions: github.ListOptions{
+			PerPage: 100,
+		},
+	}
+
+	// get all pages of results
+	var allUsers []string
+
+	for {
+		users, resp, err := mc.client.Organizations.ListMembers(mc.ctx, org, opt)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to list members of org %s page %d", org, opt.Page)
+		}
+		for _, u := range users {
+			allUsers = append(allUsers, u.GetLogin())
+		}
+		if resp.NextPage == 0 {
+			break
+		}
+		opt.Page = resp.NextPage
+	}
+
+	return allUsers, nil
+}
+
+func (mc *GitHubMembershipContext) ListTeamMembers(org, teamName string) ([]string, error) {
+	opt := &github.TeamListTeamMembersOptions{
+		ListOptions: github.ListOptions{
+			PerPage: 100,
+		},
+	}
+
+	team, _, err := mc.client.Teams.GetTeamBySlug(mc.ctx, org, teamName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Unable to get information for team %s/%s", org, team)
+	}
```

Comment:
This should take advantage of the team ID caching used in `IsTeamMember`, probably by extracting it into a new shared function.